.github/workflows/test-python.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: Test Python
```

Comment:
I'm somewhat partial to doing this in the devcontainer workflow, but I'm not going to block merging this on that.